### PR TITLE
VTable - Add rowClick event to content rows

### PR DESCRIPTION
**Pull request by lindsaykwardell**
Adds a new event (rowClick) to content rows in VTable. Returns the data from the clicked row.

I looked through the code that creates cItems, and I wasn't sure if the item being rendered to the table would be the correct solution, but I didn't see a better solution. Happy to keep working on this if it looks good to you.

I left out a cellClick event because I wasn't sure what to return in the event.

### CODE REVIEW

File: src/components/VTable/VTable.vue [hunk 1]
```diff
@@ -75,7 +75,7 @@
         </thead>
         <tbody>
           <slot v-bind="{ items: cItems, ...$data, perPage }">
-            <tr v-for="(item, index) in cItems" :key="item.id">
+            <tr v-for="(item, index) in cItems" :key="item.id" @click="$emit('rowClick', item)">
```
Comment (Stegosource) [inline src/components/VTable/VTable.vue]: Think the event name should be "click:row"?

Comment (Stegosource) [reply src/components/VTable/VTable.vue]: I'm not even sure you can name it that. But thinking it might follow Vue standards more closely.

Comment (lindsaykwardell) [reply src/components/VTable/VTable.vue]: I like that more. I don't see why it wouldn't work in Vue. I'll do a test and resubmit my request if it works.

**Comment by lindsaykwardell** [comment]
Per discussion, updated rowClick to click:row. Tested this event and it can be properly fired and subscribed to.

**Comment by Stegosource** [comment]
Merged! Thanks @lindsaykwardell. I don't have auto-publishing setup yet, so it will have to wait until my next release, but I have some cool stuff in the works!

**Comment by lindsaykwardell** [comment]
No worries, thank you!

**Pull request merged**